packages/web/server/queries/complex/pools/market.ts

```diff
@@ -23,13 +23,12 @@ export function getCachedPoolMarketMetricsMap(): Promise<
   return cachified({
     cache: metricPoolsCache,
     key: "pools-metrics-map",
-    ttl: 1000 * 10, // 10 seconds
-    staleWhileRevalidate: 1000 * 25, // 25 seconds, edge function run time limit
-    getFreshValue: async () => {
+    ttl: 1000 * 60 * 5, // 5 mins
```

Comment:
can you add some code comments here explaining why you're doing this

Reply:
Changed in https://github.com/osmosis-labs/osmosis-frontend/pull/2857/commits/6f559bbcfb89302606f8a8c8ee3268efd08fc265